e2e2/test/cases/neuron/main_test.go

```diff
@@ -0,0 +1,80 @@
+package nvidia
+
+import (
+	"context"
+	_ "embed"
+	"flag"
+	"log"
+	"os"
+	"slices"
+	"testing"
+	"time"
+
+	fwext "github.com/aws/aws-k8s-tester/e2e2/internal/framework_extensions"
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/e2e-framework/klient/wait"
+	"sigs.k8s.io/e2e-framework/pkg/env"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+)
+
+var (
+	testenv         env.Environment
+	neuronTestImage *string
+)
+
+var (
+	//go:embed manifests/k8s-neuron-device-plugin-rbac.yml
+	neuronDevicePlugiRbacManifest []byte
+	//go:embed manifests/k8s-neuron-device-plugin.yml
+	neuronDevicePluginManifest []byte
+)
+
+func TestMain(m *testing.M) {
+	neuronTestImage = flag.String("neuronTestImage", "", "image for neuron single node test")
```

Comment:
is this been used anywhere?

Reply:
Yes in here: e2e2/test/cases/neuron/manifests/single-node-test-neuronx.yaml